pkg/encoding/encoding.go

```diff
@@ -0,0 +1,22 @@
+package encoding
+
+import (
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
```

Comment:
please use `"github.com/ghodss/yaml"` to do parsing and I think it might be the problem